committee/committee.go

```diff
@@ -227,13 +230,15 @@ func (ec *committee) Start(ctx context.Context) (err error) {
 		for {
 			select {
 			case <-ec.terminate:
-				ec.terminate <- true
```

Comment:
don't understand this

in current code, ec.terminate is unbuffered channel, the goroutine will block here and won't return